### ISSUE TITLE
Can't fix 'Site can't be reached error' ERR_UNSAFE_PORT

### ISSUE THREAD

**Issue by zoenvisje** [corvideon/InstantWP#107]
Hi all,

I installed InstantWP and tried everything that's on the FAQ https://github.com/webtoolsgroup/InstantWP/wiki/Troubleshooting-InstantWP-start-up-issues but I can't get the frontpage or admin to start. I get 'site can't be reached' ERR_UNSAFE_PORT.

So it looks like it has something to do with the port that InstantWP uses (10080). Is there a fix? I found this https://www.oneminutevideotutorials.com/2021/06/21/how-to-keep-using-port-10080-after-many-browsers-block-it-as-an-unsafe-port/ , I prefer not switching to Firefox. 

Thanks in advance

**Comment by Giroud-fr** [comment]
I have the exact same problem here. I can't find a solution anywhere :(

**Comment by Giroud-fr** [comment]
with the solution described it works on firefox, but I would like to stay on Chrome too

**Comment by theatredays** [comment]
Also ran into this issue after installing InstantWP for the first time today. Any way to change the port InstantWP uses?

**Comment by essexgit** [comment]
Firefox and Edge are the same - I tried modifying config file to different port offsets - but I get the ERR_CONNECTION_REFUSED. Firefox blocked 10080 in Nov 2020 Chrome in March 2021.
I've tried turning Windows Internet Information services off&on to test conflict.

**Comment by skreemascloset** [comment]
https://www.oneminutevideotutorials.com/2021/06/21/how-to-keep-using-port-10080-after-many-browsers-block-it-as-an-unsafe-port/  worked for me

**Comment by essexgit** [comment]
> https://www.oneminutevideotutorials.com/2021/06/21/how-to-keep-using-port-10080-after-many-browsers-block-it-as-an-unsafe-port/ worked for me

Yes, you can open the port, but it was closed because of attacks.

I've moved away from WAmp, there are other, safe solutions.

**Comment by popsy-blue** [comment]
See also the answer/proposed workaround in issue #103 .

**Comment by codewizard13** [comment]
> > https://www.oneminutevideotutorials.com/2021/06/21/how-to-keep-using-port-10080-after-many-browsers-block-it-as-an-unsafe-port/ worked for me
> 
> Yes, you can open the port, but it was closed because of attacks.
> 
> I've moved away from WAmp, there are other, safe solutions.

What have you moved to? Please share suggestions for safer solutions so the rest of us can benefit
